config/config.go

```diff
@@ -130,6 +133,35 @@ func (c Config) GetPodGCDeleteDelayDuration() time.Duration {
 	return c.PodGCDeleteDelayDuration.Duration
 }
 
+func (c Config) Validate(input_url string) error {
+	u, err := url.Parse(input_url)
+	if err != nil {
+		return err
+	}
+	if u.Scheme == "javascript" {
+		return fmt.Errorf("detect javascript link: %s", input_url)
+	}
+	return nil
+}
+
+func (c Config) HTMLEscape(unescaped string) string {
+	return html.EscapeString(unescaped)
+}
+
+func (c *Config) Sanitize() {
```

Comment:
I think we should report an error on invalid links

Reply:
do you mean the argo server should log and exit 
instead of just log
when encountering link validation error?

Reply:
correct, error and exit